File: src/kyu_7/number_of_decimal_digits/cpp/number_of_decimal_digits_test.cpp
Describe(sample_tests)
{
    It(test_digits)
    {
        Assert::That(digits(5ull), Equals(1));
        Assert::That(digits(12345ull), Equals(5));
        Assert::That(digits(9876543210ull), Equals(10));
    }
};

File: src/kyu_7/number_of_decimal_digits/csharp/number_of_decimal_digits_test.cs
namespace Solution {
  using NUnit.Framework;
  using System;

  [TestFixture]
  public class SampleTests
  {
    [Test]
    public void Digits()
    {
      Assert.AreEqual(1,DecTools.Digits(5ul));
      Assert.AreEqual(5,DecTools.Digits(12345ul));
      Assert.AreEqual(10,DecTools.Digits(9876543210ul));
    }
  }
}
